website/content/docs/secrets/kv/kv-v2.mdx

```diff
@@ -183,7 +188,7 @@ allows for writing keys with arbitrary values.
 1. Read arbitrary data:
```

Comment:
numbering here makes no sense to me (seems to be per section in other parts of this doc?)